misc/acrn-config/config_app/views.py

```diff
@@ -452,7 +452,13 @@ def upload_board_info():
                 for generic_name in os.listdir(os.path.join(config_path, 'generic')):
                     generic_file = os.path.join(config_path, 'generic', generic_name)
                     if os.path.isfile(generic_file):
-                        copyfile(generic_file, os.path.join(config_path, board_type, generic_name))
+                        new_file = os.path.join(config_path, board_type, generic_name)
+                        copyfile(generic_file, new_file)
+                        xml_config = XmlConfig(os.path.join(current_app.config.get('CONFIG_PATH'),
+                                                            board_type))
+                        xml_config.set_curr(generic_name[:-4])
```

Comment:
what's the last 4 elements to skip?

Reply:
The generic_name is the configuration xml name like "industry.xml". xml_config.set_curr() shall accept the file name without suffix name as the parameter, so need to skip the last 4 characters.